app/Policies/UserPolicy.php

```diff
@@ -9,16 +9,27 @@ class UserPolicy
 {
     use HandlesAuthorization;
 
+    /**
+     * Determine whether the user can view any models.
+     *
+     * @param  \App\User  $user
+     * @return mixed
+     */
+    public function viewAny(?User $user)
+    {
+        return true;
+    }
+
     /**
      * Determine whether the user can view the model.
      *
      * @param  \App\User  $user
      * @param  \App\User  $model
      * @return mixed
      */
-    public function view(User $user, User $model)
+    public function view(?User $user, User $model)
     {
-        //
+        return true;
```

Comment:
It seems like this one should also be `return $user->is($model);`, as I can't think of any valid reason for which any (non-admin) user should be able to view a different user's model.

Reply:
Yeah, that's what I thought. Just seem to have read somewhere it also gets called for `view` (show) but that sounds really silly. I'll make a test for this.

Reply:
Oh, and it should be just `return false`